playground/Elasticseach/Elasticseach.ApiService/Program.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Elasticsearch.Net;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+
+builder.Services.AddSingleton<ElasticLowLevelClient>((sp) =>
```

Comment:
This should be encapsulated in a component right? See https://github.com/dotnet/aspire/tree/main/src/Components#readme

Reply:
@davidfowl 
for encapsulation code in a component, there are multiple packages like Nest ,Elasticsearch.Net and Elastic.Clients.Elasticsearch. The two first packages are deprecated and only **Elastic.Clients.Elasticsearch** is currently active. 

Elastic.Clients.Elasticsearch package is not in the package sources. only Elasticsearch.Net is available but already deprecated.

So, Isn't that a bad approach to using a deprecated client?
I think it's better Elastic.Clients.Elasticsearch package added in package sources.

Reply:
Sure. While we're looking for the right client, you can change the nuget.config to add NuGet.org so you can pull it in. We'll mirror the package once we decide what to use.

 Elastic.Clients.Elasticsearch seems like the one!

https://www.elastic.co/guide/en/elasticsearch/client/net-api/current/installation.html

Reply:
Running the package import for `8.14.0` of `Elastic.Clients.Elasticsearch` now.

Reply:
Its imported.